cw/tvla.py

```diff
@@ -488,6 +488,32 @@ def run_tvla(ctx: typer.Context):
         # Amount of tolerable deviation from average during filtering.
         num_sigmas = 3.5
 
+        # Slice of wave file
+        # these options are only tested for otbn
+        if (cfg["mode"] != "otbn" and "sample_start" in cfg and cfg["sample_start"] is not None):
+            raise RuntimeError('Option "sample_start is only supported for otbn!')
+        if (cfg["mode"] != "otbn" and "num_samples" in cfg and cfg["num_samples"] is not None):
+            raise RuntimeError('Option "num_samples is only supported for otbn!')
+
+        if (cfg["mode"] != "otbn" or cfg["sample_start"] is None):
+            sample_start = 0
+        else:
+            sample_start = cfg["sample_start"]
```

Comment:
```suggestion
        sample_start = 0
        if ("sample_start" in cfg and cfg["sample_start"] is not None):
            sample_start = cfg["sample_start"]
```

Currently, we will get an error if `sample_start:` is missing in the `.yaml` configuration file. This way it will be automatically set to 0 if not specified.
Similar can be done `num_samples`.
I think that after that we can just remove the `RuntimeError` checks and the feature will work for all modes.

Reply:
Good idea. I simply didn't  have time to check the other modes, but sounds reasonable.